Makefile

```diff
@@ -11,11 +11,11 @@ install: thunderhub.s9pk
 	appmgr install thunderhub.s9pk
 
 thunderhub.s9pk: manifest.yaml config_spec.yaml config_rules.yaml image.tar instructions.md
-	appmgr -vv pack $(shell pwd) -o thunderhub.s9pk
-	appmgr -vv verify thunderhub.s9pk
+	sudo $(shell which embassy-sdk) pack
+	sudo $(shell which embassy-sdk) verify thunderhub.s9pk
```

Comment:
the install command can be deleted. instead, update the `app-id.s9pk` step below, like [here](https://github.com/Start9Labs/cups-wrapper/pull/1/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R19)

there needs to be both a `embassy-sdk pack` and `embassy-sdk verify` step, with verify being the final stage of `all`